web/packages/teleterm/src/ui/TopBar/AdditionalActions.tsx

```diff
@@ -231,4 +254,13 @@ const StyledListItem = styled(ListItem)`
   gap: ${props => props.theme.space[3]}px;
   padding: 0 ${props => props.theme.space[3]}px;
   border-radius: 0;
+
+  &:disabled {
+    cursor: default;
+    color: ${props => props.theme.colors.buttons.textDisabled};
```

Comment:
WDYT about using `props.theme.colors.text.disabled` here? Opacity of `buttons.textDisabled` is 0.3 while for `text.disabled` it is 0.36 (which should be our default 'disabled' opacity looking at Figma). DevTools shows quite low contrast ratio for that text, it would be nice if we could increase it at least a little.

Reply:
Good point, it also makes more sense semantic-wise.

| Before | After |
| ---| ---
| ![before](https://github.com/gravitational/teleport/assets/27113/817dfb2d-9b57-4ea8-a807-c2ee361a02f7) | ![after](https://github.com/gravitational/teleport/assets/27113/e7b2e944-4dd2-4dbd-8c45-c33dbe5161f3) |